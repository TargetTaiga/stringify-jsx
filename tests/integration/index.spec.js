const fs = require('fs');
const path = require('path');
const stringifyJsx = require('../..');

it('should transform basic JSX', () => {
    const { code } = stringifyJsx(fs.readFileSync(path.join(__dirname, 'input.js')).toString());
    expect(code).toMatchSnapshot();
});