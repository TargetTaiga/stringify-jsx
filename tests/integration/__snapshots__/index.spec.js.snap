// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should transform basic JSX 1`] = `"const html = \`<div>\${'Hello ' + ' World!'}</div>\`;"`;
